content-src/components/DiscoveryStreamComponents/TopSites/TopSites.jsx

```diff
@@ -4,14 +4,74 @@
 
 import { connect } from "react-redux";
 import { TopSites as OldTopSites } from "content-src/components/TopSites/TopSites";
+import { TOP_SITES_MAX_SITES_PER_ROW } from "common/Reducers.jsm";
 import React from "react";
 
 export class _TopSites extends React.PureComponent {
+  insertSpocContent(TopSites, data, promoAlignment) {
+    if (
+      !TopSites.rows ||
+      TopSites.rows.length === 0 ||
+      !data.spocs ||
+      data.spocs.length === 0
+    ) {
+      return null;
+    }
+
+    const topSites = [...TopSites.rows];
+    const [topSiteSpoc] = data.spocs;
```

Comment:
My understanding spocs data passed to TopSites can have more than one spocs @ScottDowne is that correct?

Reply:
Ah, I'm destructuring the first element here, but to your other comment I should find the first non organic SPOC